src/test/authService.refreshToken.spec.ts

```diff
@@ -1,41 +1,49 @@
 import { expect } from 'chai'
 import * as sinon from 'sinon'
+import * as jwt from 'jsonwebtoken'
 
 import { AuthService } from '../apiServices/AuthService'
-import { JwtTokensInterface } from '..'
 
 const postRequest = require('../http/index')
-const errorHandler = require('../util/errorHandler')
 
 describe('refreshToken', () => {
-    const tokens: JwtTokensInterface = {
-        access: 'access',
-        refresh: 'refresh',
+    const accessToken = {
+        jti: '',
+        iss: '',
+        exp: Math.floor(new Date().getTime() / 1000) + 60 * 20,
+        userId: '',
+        iat: 0,
     }
 
     let postRequestStub: any
-    let errorHandlerStub: any
     let setCredentialsStub: any
 
     let AUTH
     beforeEach(() => {
         postRequestStub = sinon.stub(postRequest, 'postRequest')
-        errorHandlerStub = sinon.stub(errorHandler, 'errorHandler')
 
         AUTH = new AuthService('user', 'password')
+        AUTH.jwtTokens = {
+            access: jwt.sign(accessToken, 'secret'),
+            refresh: 'refresh',
+        }
+
         setCredentialsStub = sinon.stub(AUTH, 'setCredentials')
```

Comment:
I don't think we need to stub setCredentials, but we can probably add something like
```
expect(AUTH.jwtTokens.access).to.equal("expectedtoken")
expect(AUTH.jwtTokens.refresh).to.equal("expectedrefreshtoken")
```
to the end of the "returns a set of tokens" test.